src/classes/Passage/Passage.ts

```diff
@@ -1,224 +1,60 @@
-        let passageAuthToken;
-        for (const cookie of cookies) {
-            const sepIdx = cookie.indexOf('=');
-            if (sepIdx === -1) {
-                continue;
-            }
-            const key = cookie.slice(0, sepIdx).trim();
-            if (key !== 'psg_auth_token') {
-                continue;
-            }
-            passageAuthToken = cookie.slice(sepIdx + 1).trim();
-            break;
-        }
-
-        if (passageAuthToken) {
-            const userID = await this.validAuthToken(passageAuthToken);
-            if (userID) return userID;
-            else {
-                throw new PassageError('Could not validate auth token. You must catch this error.');
-            }
-        } else {
-            throw new PassageError(
-                "Could not find authentication cookie 'psg_auth_token' token. You must catch this error.",
-            );
-        }
-    }
-
-    /**
-     * @deprecated Use Passage.auth.validateJwt instead.
-     * Determine if the provided token is valid when compared with its
-     * respective public key.
-     *
-     * @param {string} token Authentication token
-     * @return {string} sub claim if the jwt can be verified, or Error
-     */
-    async validAuthToken(token: string): Promise<string | undefined> {
-        return this.auth.validateJwt(token);
-    }
-
-    /**
-     * @deprecated Use Passage.auth.createMagicLink instead.
-     * Create a Magic Link for your app.
-     *
-     * @param {CreateMagicLinkRequest} args options for creating a MagicLink.
-     * @return {Promise<MagicLink>} Passage MagicLink object
-     */
-    async createMagicLink(args: CreateMagicLinkRequest): Promise<MagicLink> {
-        try {
-            const magicLinksApi = new MagicLinksApi(this._apiConfiguration);
-            const response = await magicLinksApi.createMagicLink({
-                appId: this.appId,
-                createMagicLinkRequest: args,
-            });
-
-            return response.magicLink;
-        } catch (err) {
-            if (err instanceof ResponseError) {
-                throw await PassageError.fromResponseError(err, 'Could not create a magic link for this app');
-            }
-            throw err;
-        }
-    }
-
-    /**
-     * @deprecated Passage.auth.validateJwt will validate the JWT audience automatically.
-     * Get App Info about an app
-     *
-     * @return {Promise<AppInfo>} Passage App object
-     */
-    async getApp(): Promise<AppInfo> {
-        try {
-            const client = new AppsApi(this._apiConfiguration);
-            const response = await client.getApp({
-                appId: this.appId,
-            });
-
-            return response.app;
-        } catch (err) {
-            if (err instanceof ResponseError) {
-                throw await PassageError.fromResponseError(err, 'Could not fetch app');
-            }
+    private configureApi(config?: ConfigurationParameters): Configuration {
```

Comment:
moved this from the root `utils` directory since it's only used in this class's constructor